packages/suite/src/views/wallet/send/Outputs/Address.tsx

```diff
@@ -158,39 +168,56 @@ export const Address = ({ output, outputId, outputsCount }: AddressProps) => {
         }
     }, [amountInputName, composeTransaction, dispatch, inputName, setValue, symbol]);
 
-    const getValidationButtonProps = (): InputErrorProps['button'] => {
+    const getInputErrorProps = (): {
+        learnMoreUrl?: InputErrorProps['learnMoreUrl'];
+        buttonProps?: InputErrorProps['buttonProps'];
+    } => {
         switch (addressError?.type) {
             case 'deprecated':
-                if (addressDeprecatedUrl) {
+                return {
+                    learnMoreUrl: addressDeprecatedUrl ? URLS[addressDeprecatedUrl] : undefined,
+                };
+            case 'evmchecks':
+                if (!checkAddressCheckSum(address)) {
```

Comment:
I get a deprecation warning here, maybe it's better to fix it now rather than later:

> Will be removed in next release. Please use `web3-validator` package instead.

Reply:
Note - doesn't have to be in this PR

Reply:
Created an issue for this https://github.com/trezor/trezor-suite/issues/15989